ds_judgements_public_ui/templates/includes/structured_search_additional_inputs.html

```diff
@@ -2,16 +2,17 @@
   <div class="structured-search__multi-fields-panel">
     <div class="structured-search__specific-field-container">
       <label for="neutral_citation" class="structured-search__limit-to-label">Neutral citation</label>
+      <p class="structured-search__help-text" id="neutral_citation-help-text">For example [2021] EWCA Crim 1785</p>
```

Comment:
Like we discussed on slack, it could be worth creating a shared input element, something like

```
<div class="structured-search__multi-fields-panel">
    <div class="structured-search__specific-field-container">
      <label for={{ input_id }} class="structured-search__limit-to-label">{{ input_name }}</label>
      <p class="structured-search__help-text" id={{ input_name_id }} + "-help-text">{{ help_text }}</p>
      <input class="structured-search__limit-to-input"
             id={{ input_id }}
             name={{ input_name_id }}
             type={{type}}
             value="{{ context.{{ input_name_id }} }}"
             aria-describedby={{ input_name_id }} + "-help-text" />
    </div>
```

as a macro or template saved in, for example, `components/input.html`

and then use it like

```
{% include 'components/input.html' with 
    input_id='neutral_citation', 
    input_name='Neutral citation',
    input_name_id='neutral_citation',
    type='text',
    help_text='For example [2021] EWCA Crim 1785',
    context=context
%}
```